packages/synapse-react-client/src/components/DataGrid/columns/AutocompleteColumn.tsx

```diff
@@ -76,13 +91,20 @@ function AutocompleteCell({
 
 export type AutocompleteColumnProps = {
   choices: AutocompleteOption[]
+  colType?: JSONSchema7Type
 }
 
-export function autocompleteColumn({ choices }: AutocompleteColumnProps) {
+export function autocompleteColumn({
+  choices,
+  colType,
+}: AutocompleteColumnProps): Partial<Column> {
   return {
-    component: (props: Omit<AutocompleteCellProps, 'choices'>) => (
-      <AutocompleteCell {...props} choices={choices} />
-    ),
+    component: ((props: Omit<AutocompleteCellProps, 'choices'>) => (
+      <AutocompleteCell {...props} choices={choices} colType={colType} />
+    )) as CellComponent,
+    // If we update our enums to support labels, then we can update copy to copy the label and paste to lookup the mapping from label -> value
+    copyValue: ({ rowData }) => rowData,
+    pasteValue: ({ value }) => value,
```

Comment:
add copy/paste support (required to make drag work)

if we support custom labels for enums, we would need to add additional logic here

Reply:
we may need something similar for boolean columns as well.